build/runTest.sh

```diff
@@ -19,3 +19,5 @@ then
 else
     sh -c "$TEST"
 fi
+
+exit $?
```

Comment:
I'm still working on it right now, it should but it is strange that if composer failed the test was green